src/mono/wasm/build/WasmApp.Native.targets

```diff
@@ -194,6 +194,7 @@
       <_EmccCommonFlags Include="$(_DefaultEmccFlags)" />
       <_EmccCommonFlags Include="$(EmccFlags)" />
       <_EmccCommonFlags Include="-s EXPORT_ES6=1" />
+      <_EmccCommonFlags Include="-s DYNAMIC_EXECUTION=0" />
```

Comment:
Can we put in a comment that describes why this is here and what it does? I assume it causes emcc to produce CSP-safe code, but does it have side effects we care about other than that?

Reply:
The doc is here https://emscripten.ru/docs/api_reference/emscripten.h.html#c.emscripten_run_script
I think we actualy don't need to set `DYNAMIC_EXECUTION` and I will revert it.